scipy/stats/_hypotests.py

```diff
@@ -995,18 +996,22 @@ def _binomial_maximisation_of_p_value_with_nuisance_param(
         nuisance_power_n_minus_x1_x2 = log_1_minus_nuisance * (n - x1_sum_x2)
         nuisance_power_n_minus_x1_x2[(x1_sum_x2 == n)[:, :, 0]] = 0
 
-        tmp_values_arr = np.exp(
-            x1_sum_x2_log_comb
-            + nuisance_power_x1_x2
-            + nuisance_power_n_minus_x1_x2
+        tmp_log_values_arr = (
+                x1_sum_x2_log_comb
+                + nuisance_power_x1_x2
+                + nuisance_power_n_minus_x1_x2
```

Comment:
This is over-indented
```suggestion
            x1_sum_x2_log_comb
            + nuisance_power_x1_x2
            + nuisance_power_n_minus_x1_x2
```